src/frontend/src/tables/company/CompanyTable.tsx

```diff
@@ -157,16 +157,11 @@ export function CompanyTable({
           params: {
             ...params
           },
+          modelType: ModelType.company,
           tableFilters: tableFilters,
           tableActions: tableActions,
           enableDownload: true,
-          rowActions: rowActions,
-          onRowClick: (row: any) => {
-            if (row.pk) {
-              let base = path ?? 'company';
-              navigate(`/${base}/${row.pk}`);
-            }
-          }
```

Comment:
Q: Why is the direct link removed?

Reply:
Ah good catch. The "open link in new tab by holding control" wasn't working for this table, but I forgot that the link can be passed in. I'll fix this.